### ISSUE TITLE
@IcingTaskManage

### ISSUE THREAD

**Issue by vegetablebrain** [linuxmint/cinnamon-spices-applets#1184]
Have you read the guidelines for contributing?

Before opening an issue, please read the guidelines for contributing linked above. Issues with invalid formats, or issues not mentioning the maintainer of the applet, may not get a response, and eventually will be closed. You can find the author by visiting the applet's page on the Spices website (https://cinnamon-spices.linuxmint.com/), or going to the applet's directory on this repository, and checking the the author field in info.json. If it is set to none in info.json, or there is no author listed on the site, then this step is not necessary.

[Notify author of applet]: @<GithubAuthor>

[Tips]: <> For bug reports, it helps the maintainer of the applet you are reporting an issue on when you give them some environmental information for debugging. This can help them figure out what is wrong with it more quickly. Some things you can include in your bug report are the:

 - Distribution (In a terminal run "lsb_release -a".)
 - Cinnamon version (In a terminal run "cinnamon --version".)
 - Applet version (Right-click the applet, then click "About...", or go to its directory and include the version/last-edited values in metadata.json.)
 - ~/.cinnamon/glass.log (Use pastebin or a small snippet that's relavent to the issue.)
 - ~/.xsession-errors (If you have one.)

[Issue]: <> (Describe your issue below.)

**Comment by vegetablebrain** [comment]
Tried to install to panel from "applets" app.  There was a problem.  From glass log:

error t=2017-08-28T19:26:30.764Z [Applet "IcingTaskManager@json"]: File not found: /home/me/.local/share/cinnamon/applets/IcingTaskManager@json/applet.js
error t=2017-08-28T19:26:30.764Z Could not load applet IcingTaskManager@json

LSB Version:	core-2.0-amd64:core-2.0-noarch:core-3.0-amd64:core-3.0-noarch:core-3.1-amd64:core-3.1-noarch:core-3.2-amd64:core-3.2-noarch:core-4.0-amd64:core-4.0-noarch:core-4.1-amd64:core-4.1-noarch:security-4.0-amd64:security-4.0-noarch:security-4.1-amd64:security-4.1-noarch
Distributor ID:	LinuxMint
Description:	Linux Mint 17.3 Rosa
Release:	17.3
Codename:	rosa

Cinnamon 2.8.8

I downloaded this version about 7pm GMT and the site (cinnamon spices) said it was 8hrs old.  Today is mon 28 Aug 2017.

So, basically, it won't add to the panel.

**Comment by vegetablebrain** [comment]
Hey, I haven't used this system before.   Give me a break, huh?   I think the description is clear.

**Comment by jaszhix** [comment]
Did you download it from the website? If so, delete ```~/.local/share/cinnamon/applets/IcingTaskManager@json```, and install it through System Settings -> Applets.

**Comment by jaszhix** [comment]
Okay, the version directory name was incorrect. Reinstall ITM through Applet Settings, and it should work now.
